packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -229,7 +229,7 @@ export interface GetPrismaClientConfig {
    * If enabled, we disregard the generator config engineType.
    * It means that `--data-proxy` binds you to the Data Proxy.
    */
-  dataProxy: boolean
+  dataProxy?: boolean
```

Comment:
I would prefer adding `dataProxy: false` after line 62 in `packages/client/src/utils/getTestClient.ts` instead of making it optional here.

Reply:
Sure: https://github.com/prisma/prisma/pull/13875/commits/c66bb3d3e897249dcb986856c65b17f0bbe9da1e